src/providers/data_providers/businessImageProviderAggregator.ts

```diff
@@ -2,16 +2,26 @@ import { BusinessImageProvider } from './businessImageProvider';
 import { ImageProviderName } from '../../models/data_models/image';
 import { MockBusinessImageProvider } from './mockBusinessImageProvider';
 import { YelpBusinessImageProvider } from './yelpBusinessImageProvider';
+import { OpenCageApiUtils } from '../../utils/openCageApiUtils';
 import { Image } from '../../models/data_models/image';
 import { BusinessInfoInput } from '../../models/data_models/types';
+import { defaultConfig, DataSourceConfiguration } from '../../models/data_models/dataSourceConfiguration';
 
 export class BusinessImageProviderAggregator {
+	// load configs here
+	protected dataSourceConfigs: DataSourceConfiguration;
+
+	constructor() {
+		this.dataSourceConfigs =
+			(process.env.BUSINESS_API_CONFIG && JSON.parse(process.env.BUSINESS_API_CONFIG)) || defaultConfig;
+	}
```

Comment:
We might even go a step further to pass data source config in as a constructor parameter and leave config to the responsibility of the code that puts all our providers together